server/rest/dataset.py

```diff
@@ -349,6 +349,14 @@ def promoteItemToDataset(self, item, params):
                 minerva_metadata['original_files'] = [{
                     'name': file['name'], '_id': file['_id']}]
                 break
+            elif 'tif' in file['exts'] and file['mimeType'] == 'image/tiff':
```

Comment:
@matthewma7 Uploading a regular tiff file as opposed to a geotiff file breaks this. We should be able to check whether the "srs" is empty  or not in the info call (GET /ktile/{id}/info).

Reply:
@dorukozturk, Got it, I see what you mean.